### DISCUSSION TITLE
Have health of Job Created by CronJob update app health

### DISCUSSION THREAD

**Discussion by phyzical** [argoproj/argo-cd#10569] [Q&A]
Hey there,

i have been searching for a few days now and i cant seem to find if this is even possible, but we have batch/CronJob objects that create batch/Jobs.

What we were hoping was that when a Job is degraded (from hitting its backofflimit) the App status would also become degraded so that we can have a notification fire off to alert us of the issue.

At first we thought adding a custom health check was the solution, which does update the heath check for Jobs specifically sadly this health does not propagate to cronjobs or the overall app.

tldr 
* can you have a CronJob Jobs health propagate to an apps overall health
* can you send notifications on specific resources i.e notify when Job degraded
